tests/utils/constants.ts

```diff
@@ -1,5 +1,12 @@
 import initialContractState from './initial-state.json';
 
+export const DEFAULT_FOUNDATION_ACTION_ACTIVE_STATUS = 'active';
+export const DEFAULT_FOUNDATION_ACTION_PASSED_STATUS = 'passed';
+export const DEFAULT_FOUNDATION_ACTION_FAILED_STATUS = 'failed';
+export const DEFAULT_FOUNDATION_DELAYED_EVOLVE_COMPLETED_STATUS = 'evolved';
```

Comment:
these are already available in the tests - we `export * from '../../constants.ts` below - meaning all constants in the contract are already available. these constants only include anything additional related specifically to tests